templates/base.html.twig

```diff
@@ -37,12 +37,18 @@
                 <li class="px-2 nav-item">
                     <a class="nav-link" href="{{ path("home") }}#tricks">Les Tricks</a>
                 </li>
+                {% if is_granted('ROLE_USER') %}
```

Comment:
And if is granted ROLE ADMIN ?

Reply:
Every logged user has role "ROLE_USER" i use this to check if the user is already logged